clients/privacy-center/packages/fides-consent/README.md

```diff
@@ -59,7 +85,27 @@ if (Fides.consent.data_sales) {
 }
 ```
 
-## fides-consent.mjs & fides-consent.d.ts
+### Shopify
+
+To integrate with Shopify's [Consent Tracking API](https://shopify.dev/api/consent-tracking?shpxid=7e81a186-C696-4E23-F327-E7F38E5FF5EE#consent-collection),
+call `Fides.shopify(options)`, where options is an with the following properties:
+
+- `tracking`: The only consent option Shopify currently supports. Refer to their [visitor tracking](https://shopify.dev/api/consent-tracking#visitor-tracking) documentation.
+
+For example, with the default Privacy Center configuration:
+
+```html
+<head>
+  <!-- The script can be loaded in the store's theme, or in a custom pixel. -->
+  <script src="example.com/privacy-center/fides-consent.js"></script>
+  <script>Fides.shopify({ tracking: Fides.consent.data_sales })</script>
```

Comment:
Note that I'm still using/recommending explicit function calls for this integrations. I don't have enough examples to know how the [magic configuration](https://ethyca.slack.com/archives/C04774UV24D/p1673035044508019) idea would work.